src/main/java/com/google/sps/data/Contest.java

```diff
@@ -42,7 +43,7 @@ public static Builder builder() {
   public abstract static class Builder {
     public abstract Builder setName(String name);
 
-    public abstract Builder setCandidates(Set<Long> candidates);
+    public abstract Builder setCandidates(Collection<Long> candidates);
```

Comment:
Why are we changing this to Collection? I think we want to leave it as a Set, or else there may be bugs involving repeated candidate ids, no? (Maybe autovalue is smart enough to convert it correctly, but we shouldn't depend on that).

Reply:
I ran into a bug where getProperty in ContestServlet was returning an ArrayList that couldn't be casted to a Set so I decided to use Collections instead as a sort of catch all. Here are the errors I ran into while using HashSet instead of Collection:
![Screenshot 2020-08-06 at 11 27 42 AM - Display 1](https://user-images.githubusercontent.com/41168973/89550785-15fc8580-d7d8-11ea-828d-183bde4399ad.png)
![Screenshot 2020-08-06 at 11 27 42 AM - Display 2](https://user-images.githubusercontent.com/41168973/89550795-18f77600-d7d8-11ea-944c-6d29ebe754e0.png)

Reply:
Alternatively you can create a set from the ArrayList that's returned, e.g. `ImmutableSet.copyOf(contestEntity.getProperty(...))`